README.md

```diff
@@ -52,17 +52,25 @@ var api = RiotApi.GetInstance("YOUR_API_KEY",
   yourRateLimitPer10s, yourRateLimitPer10m);
 ```
 
-To get a summoner:
+To get basic data of a summoner:
 ```c#
 try
 {
-  var summoner = api.GetSummonerByName(Region.euw, "StopOFlop");
+  var summoner = api.Summoner.GetSummonerByNameAsync(Region.euw, "StopOFlop").Result;
+  var name = summoner.Name;
+  var level = summoner.Level;
+  var accountId = summoner.AccountId;
 }
 catch (RiotSharpException ex)
 {
   // Handle the exception however you want.
 }
 ```
+All classes were separated according to their tasks. The following classes can be distinguished:  
+> <i>Champion League Match Spectator Status Summoner ChampionMastery StaticData ThirdParty  
+> [Wiki](http://benfradet.github.io/RiotSharp/api/RiotSharp.IRiotApi.html)
+</i>
```

Comment:
What do you mean by this paragraph?

Reply:
In my opinion, the whole logic of interfacing interfaces and classes is divided into its tasks. This means - if we want to extract information about the player from the API - we use the Summoner class.
At least I think so after using this repository.

Reply:
ok then maybe make a markdown list

```
Each class represents an API:
- Champion
- League
- etc
```

and you can remove the link to the docs there is a link at the top of https://github.com/BenFradet/RiotSharp

Reply:
You are right. 
Done.